deployments/common/crds/appprotectdos.f5.com_dosprotectedresources.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
```

Comment:
One of the other CRD was generated with `v0.4.0` and one doesn't have the annotation? 🤔

Reply:
one crd is generated by myself (dos protected resources) using the existing tools in our repo. The other crds were added manually, either by copy pasting existing crds or generating it somewhere else. I'm not sure there's a functional difference. I've added the missing annotation anyway.